mne/preprocessing/artifact_detection.py

```diff
@@ -333,3 +334,184 @@ def _annotations_from_mask(times, art_mask, art_name):
             durations.append(times[l_idx[-1]] - times[l_idx[0]])
         desc.append(art_name)
     return Annotations(onsets, durations, desc)
+
+
+@verbose
+def annotate_break(raw, events=None, min_duration=15., dist_to_previous=5.,
+                   dist_to_next=5., ignore=('bad', 'edge'), *, verbose=None):
+    """Create `~mne.Annotations` for breaks in an ongoing recording.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The continuous data to analyze.
+    events : array, shape (n_events, 3)
+        An events array. If ``None`` (default), operate based solely on the
+        annotations present in ``raw``. If an array, ignore any annotations
+        in the raw data, and only operate based on these events.
+    min_duration : float
+        The minimum time span in seconds between the offset of one and the
+        onset of the subsequent annotation (if ``events`` is ``None``) or
+        between two consecutive events (if ``events`` is an array) to consider
+        this period a "break". Defaults to 15 seconds.
+    dist_to_previous, dist_to_next : float
+        Specifies how far the "break" annotation extends towards the beginning
+        and end of the time period between the two annotations or events
+        spanning a break. This can be used to ensure e.g. that the break
```

Comment:
```suggestion
    t_after, t_before : float
        Specifies when to start the break after the last event (or annotation) and before
        the next one. This can be used to ensure e.g. that the break
```